challenge.md

```diff
@@ -6,19 +6,33 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+We type
+git clone [ssh URL of forked git repo]
+in the Terminal app
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git diff
```

Comment:
We were looking for `git status`, but based on how the question was worded, `git diff` is a reasonable answer - I won't take off any points.